src/VisualStudio/Core/Test/Options/CSharpEditorConfigGeneratorTests.vb

```diff
@@ -0,0 +1,312 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Text
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.CodeStyle
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Roslyn.Test.Utilities
+
+Namespace Microsoft.VisualStudio.LanguageServices.UnitTests
+    <[UseExportProvider]>
+    Public Class CSharpEditorConfigGeneratorTests
+        Inherits TestBase
+
+        <ConditionalWpfFact(GetType(IsEnglishLocal))>
```

Comment:
these need Wpf?

Reply:
We can only apply one fact attribute to a test. Also note that `ConditionalWpfFact` derives from `WpfFact`.

---
In reply to: [219240596](https://github.com/dotnet/roslyn/pull/30043#discussion_r219240596) [](ancestors = 219240596)

Reply:
My question was more: Why do these need Wpf?  What about the test is wpf specific?

Reply:
Ah, correct, fixed.

---
In reply to: [219300109](https://github.com/dotnet/roslyn/pull/30043#discussion_r219300109) [](ancestors = 219300109)